### ISSUE TITLE
ckpt file doesn't match embedding_manager model

### ISSUE THREAD

**Issue by AnqiGou** [tyxsspa/AnyText#151]
In the earlier code you published, the model definition for the OCR embedding was as follows: if emb_type == 'ocr': self.proj = linear(40 * 64, token_dim). This allowed for the successful loading of the pretrained weights from the state_dict(v1-5-scratch.ckpt). However, after training with 200k samples for 15 epochs as README.md, the model's generation performance was quite poor, with a laion acc_sen of only 0.05. 

In your later code release, the OCR embedding model is redefined as follows: if emb_type == 'ocr': self.proj = nn.Sequential(zero_module(linear(40 * 64, token_dim)), nn.LayerNorm(token_dim)). However, I am unable to load the corresponding pretrained weights from the state_dict, encountering the following errors:

Missing key(s) in state_dict: "embedding_manager.proj.0.weight", "embedding_manager.proj.0.bias", "embedding_manager.proj.1.weight", "embedding_manager.proj.1.bias". 
Unexpected key(s) in state_dict: "embedding_manager.proj.weight", "embedding_manager.proj.bias".

Could the lack of normalization be the reason for the poor convergence after 15 epochs?  If so, how should I load the pretrained weights corresponding to the new definition? Specifically, does the original state_dict store weights under embedding_manager.proj.0, and are the parameters for embedding_manager.proj.1.bias zero-initialized, embedding_manager.proj.1.weights 1-initialized?

Thank you very much for your assistance!
